BlueprintUI/Sources/Layout/Aligned.swift

```diff
@@ -111,3 +111,23 @@ public struct Aligned: Element {
         }
     }
 }
+
+public extension Element {
+    /// Wraps the element in an `Aligned` element with the provided parameters.
+    ///
+    /// - parameters:
+    ///   - vertically: The vertical alignment. Defaults to `.centered`.
+    ///   - horizontally: The horizontal alignment. Defaults to `.centered`.
+    ///
+    func aligned(
+        vertically: Aligned.VerticalAlignment = .center,
```

Comment:
I don't love a default in this case because it's not clear reading the code that `.aligned()` means center-aligned. What do you think about removing the default and adding a `centered()` function if that's a common use case?

Edit: looks like there's already `centered`! Let me know what you think about removing the default

Reply:
I kept the defaults from the normal init on `Aligned`, since sometimes you only want to pass one value, and I think centered is a pretty reasonable default. Thoughts?

Reply:
Eh, what the hell, removing the default